vignettes/col_plan.Rmd

```diff
@@ -14,6 +14,283 @@ knitr::opts_chunk$set(
 )
 ```
 
-```{r setup}
+```{r setup, echo = FALSE, message = FALSE, warning = FALSE}
 library(tlang)
+library(dplyr)
 ```
+
+The purpose of the column plan (`col_plan`) is to allow the user to specify how columns in the final table should appear in regards to their labeling and ordering. Similar to the column alignment plan (`col_align_plan`), `col_plan` refers to the final columns of the table, i.e., the values found in the `column` variable(s) from the input dataset. The `col_plan` is like a glorified version of dplyr's `select` function; just like `select`, the user can add, drop, order, and/or rename columns all in one step. What sets the `col_plan` apart, however, is the ability to also specify column spanners and labels in a hierarchical fashion.
+
+The specification and behavior of the `col_plan` is dependent on the number of column variables that are provided to the tfrmt:
+
+| col_plan options:        | 1 column variable | >1 column variable                |
+| ------------------------ | ----------------- | --------------------------------- |
+| **Rename columns**       | Yes               | Yes (lowest level col only)       |
+| **Reorder columns**      | Yes               | Yes                               |
+| **Add spanning labels**  | Yes               | No (provided by upper level cols) |
+
+As shown in the table, the `col_plan` is most powerful in conjunction with the use of a single `column` variable. However, in many cases, such as when a user performs the analysis by treatment group, these "by" variables can essentially be re-purposed as data-driven spanners in the form of upper level `column` variables. 
+
+Let's look at each of these key workflows in more detail.
+
+# Option 1: A single `column` variable 
+
+Let's consider the following dataset which includes a single `column` variable:
+
+```{r}
+dat <- tribble(
+    ~group,     ~label,  ~my_col,    ~parm, ~val,
+    "g1", "rowlabel1",  "col1"  ,  "value",    1,
+    "g1", "rowlabel1",  "col2"  ,  "value",    1,
+    "g1", "rowlabel1",  "mycol3",  "value",    1,
+    "g1", "rowlabel1",  "col4"  ,  "value",    1,
+    "g1", "rowlabel1",  "mycol5",  "value",    1,
+    "g1", "rowlabel2",  "col1"  ,  "value",    2,
+    "g1", "rowlabel2",  "col2"  ,  "value",    2,
+    "g1", "rowlabel2",  "mycol3",  "value",    2,
+    "g1", "rowlabel2",  "col4"  ,  "value",    2,
+    "g1", "rowlabel2",  "mycol5",  "value",    2,
+    "g2", "rowlabel3",  "col1"  ,  "value",    3,
+    "g2", "rowlabel3",  "col2"  ,  "value",    3,
+    "g2", "rowlabel3",  "mycol3",  "value",    3,
+    "g2", "rowlabel3",  "col4"  ,  "value",    3,
+    "g2", "rowlabel3",  "mycol5",  "value",    3)
+  
+```
+
+
+## No column spanners
```

Comment:
I think a title that is more descriptive of what it is, rather than what it isnt may be more helpful for folks.
```suggestion
## Simple Column Selection
```